File: Project/Data/Models/Table.cs
namespace Project.Data.Models
{
    public class Table
    {
        public int Id { get; set; }
        public int Number { get; set; }
        public int Capacity { get; set; }
        public byte[]? QRCodeImage { get; set; } // QR Code image për tavolinën
        public ICollection<Order> Orders { get; set; }
        public ICollection<Reservation> Reservations { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedAt { get; set; }
        public string? UpdatedBy { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}

File: frontend/src/types/table.ts
export interface TableDto {
  id: number;
  number: number;
  capacity: number;
  totalOrders: number;
  qrCodeImage?: string; // Base64 string për QR code image
  createdBy: string;
  createdAt: string;
  updatedBy?: string;
  updatedAt?: string;
}

export interface TableOrderSummaryDto {
  status: string;
  totalAmount: number;
  tableNumber: number;
  items: OrderItemSummaryDto[];
}

export interface OrderItemSummaryDto {
  menuItemId: number;
  menuItemName: string;
  quantity: number;
  unitPrice: number;
  total?: number;
}
